examples/observability/README.adoc

```diff
@@ -0,0 +1,66 @@
+= Camel Quarkus Observability
+
+This example project demonstrates how to add support for metrics, health checks and distributed tracing.
+
+=== Running
+
+[source]
+----
+$ mvn clean compile quarkus:dev -DnoDeps
+----
+
```

Comment:
```suggestion
This compiles the project and starts the Quarkus tooling in the https://quarkus.io/guides/maven-tooling#development-mode[develpment mode].

To build a https://quarkus.io/guides/maven-tooling#building-a-native-executable[native executable] run the following (Docker required)

[source]
￼----
￼$ mvn clean package -Dnative
￼----

and find the native executable in the `target` directory.

```

Reply:
Thanks, I've now added this to info the README.